file_paths.md

```diff
@@ -163,160 +163,202 @@ The package will create the following services `puppet`, `mcollective`,
+            pxp-agent.conf                        # pxp-agent configuration file
+            modules *                             # stores configuration files for pxp-agent modules
+                pxp-module-puppet.conf            # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
         var *
             log *
-                pxp-agent.log                 # enabled by default
-            spool *                           # directory containing results of pxp-agent modules
+                pxp-agent.log                     # enabled by default
             run *
+            spool *                               # directory containing results of pxp-agent modules
+
+    C:\Program Files\Puppet Labs\Puppet\bin *     # symlink targets of puppet related binaries
+        environment.bat@ *                        -> C:\Program Files\Puppet labs\Puppet\puppet\bin\environment.bat
+        facter.bat@ *                             -> C:\Program Files\Puppet labs\Puppet\puppet\bin\facter.bat
+        facter_interactive.bat@ *                 -> C:\Program Files\Puppet labs\Puppet\puppet\bin\facter_interactive.bat
+        hiera.bat@ *                              -> C:\Program Files\Puppet labs\Puppet\puppet\bin\hiera.bat
+        mco.bat@ *                                -> C:\Program Files\Puppet labs\Puppet\puppet\bin\mco.bat
+        puppet.bat@ *                             -> C:\Program Files\Puppet labs\Puppet\puppet\bin\puppet.bat
+        puppet_interactive.bat@ *                 -> C:\Program Files\Puppet labs\Puppet\puppet\bin\puppet_interactive.bat
+        puppet_shell.bat@ *                       -> C:\Program Files\Puppet labs\Puppet\puppet\bin\puppet_shell.bat
+        run_facter_interactive.bat@ *             -> C:\Program Files\Puppet labs\Puppet\puppet\bin\run_facter_interactive.bat
+        run_puppet_interactive.bat@ *             -> C:\Program Files\Puppet labs\Puppet\puppet\bin\run_puppet_interactive.bat
 
-    C:\Program Files\Puppet Labs\Puppet
-        VERSION                               # puppet-agent package version
-
-    C:\Program Files\Puppet Labs\Puppet\bin *
-        environment.bat *                     # setup LOAD_PATH
-        facter.bat *                          # bat file wrappers
-        hiera.bat *
-        mco.bat *
-        puppet.bat *
-        puppet_shell.bat *                    # targets for shortcuts
-        run_facter_interactive.bat *
-        run_puppet_interactive.bat *
-
-    C:\Program Files\Puppet Labs\Puppet\facter *
-        bin *                                 # executables and dlls
+    C:\Program Files\Puppet Labs\Puppet\misc *
+        LICENSE.rtf *                             # license
+        puppetlabs.ico *                          # icon for start menu shortcut
+        puppetres.dll *                           # event log message resource dll
```

Comment:
@melissa Just wondering if we should leave ```puppetres.dll``` in the ```bin``` directory (where it is built to now).
Otherwise I will need to look at the build and how this is built.
The ```.ico``` file can live either here or bin directory, but since the only other files here are "document type", i.e. license and version list, it would probably be better to put the ```.ico`` file in ```bin```

Tx